ckanext/ontario_theme/fanstatic/common.css

```diff
@@ -1081,6 +1193,143 @@ via submission form. Copied from DS.*/
     line-height: 1.5;
   }
 }
+.ministry-contact {
+	margin-top: 28px;
+	margin-bottom: 28px;
+}
+
+.resource-item {
+  padding-top: 15px;
+	padding-bottom: 15px;
+}
+
+.resource-item-description{
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24.51px;
+  color: #1a1a1a;
+}
+.label.label-english,
+.label.label-french,
+.label.label-english_and_french {
+  background-color: #f2f2f2;
+  font-size: 20px;
+  font-weight: 700;
+  color: #1a1a1a;
+  border-radius: 2px;
+  margin-left: 10px;
+  margin-bottom: 5px;
+}
+
+.resource-item-title{
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 27px;
+  color: #000000;
+  text-decoration: none;
+}
+
+.resource-item-title:hover{
+  color: #000000;
+}
+
+.dataset-font{
+  font-size: 19px;
+  line-height: 25.6px;
+  font-weight: 400;
+}
+.right-col{
+  float: right;
+  max-width: 350px;
```

Comment:
unset max-width on small screens so the aside is full width

Reply:
This seems to have done the trick, though I'm not sure if it's what you would have done:
```
.right-col{
  float: right;
}

@media screen and (min-width: 73em) {
  .right-col {
    max-width: 350px;
  }
}
```